kolibri/utils/server.py

```diff
@@ -3,6 +3,7 @@
 import cherrypy
 from django.conf import settings
 from django.core.management import call_command
+from kolibri.content.utils import paths
 from kolibri.content.utils.annotation import update_channel_metadata_cache
 from kolibri.deployment.default.wsgi import application
```

Comment:
At some point, we should consider how `kolibri.utils` is kept clean from importing other modules. Not that it immediately poses a threat, because we can grasp these few harmless imports.

One interesting solution would be to migrate `kolibri.utils.server` to `kolibri.deployment.cherrypy` or something likewise. Because it's essentially a pluggable feature that depends on a couple of things that are not in `kolibri.utils`.